src/web/lib/components/ThemeBackgroundPicker/index.js

```diff
@@ -1,112 +1,16 @@
 import React from "react";
```

Comment:
I just renamed some files here...what used to be the ThemeBackgroundPicker is not the ThemePatternPicker